appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -423,6 +423,19 @@ public void SelectionIndex(int index) {
     }
   }
 
+  /**
+   */
```

Comment:
You can move the description below into the Javadoc and the documentation generator should take care of the rest.